sensor/common/registry/tlscheck_cache.go

```diff
@@ -84,6 +89,8 @@ func (c *tlsCheckCacheImpl) Cleanup() {
 // CheckTLS performs a TLS check on a registry or returns the result from a
 // previous check. Returns true for skip if there was a previous error.
 func (c *tlsCheckCacheImpl) CheckTLS(ctx context.Context, registry string) (secure bool, skip bool, err error) {
+	metrics.IncrementTLSCheckCount()
```

Comment:
I think a previous commit counted the number of cache hits. That may still be useful so we can compare how many cache hits we get vs the total number of calls

Reply:
This 'new' way does allow for calculating hits and is more accurate then in the previous PR ([explanation here](https://github.com/stackrox/stackrox/pull/12379#discussion_r1717353976)), I had also added an example of how to do this in the PR description, TL;DR:

Cache | How
--- | ---
Total Requests | `rox_sensor_registry_store_tls_check_count`
Miss | `rox_sensor_registry_store_tls_check_duration_seconds_count`
Hit |  `rox_sensor_registry_store_tls_check_count` - `rox_sensor_registry_store_tls_check_duration_seconds_count`